ppb/sprites.py

```diff
@@ -209,43 +246,40 @@ def center(self) -> Vector:
         return self.position
 
     @center.setter
-    def center(self, value: Sequence[float]):
-        x = value[0]
-        y = value[1]
-        self.position.x = x
-        self.position.y = y
+    def center(self, value: ppb_vector.vector2.VectorLike):
```

Comment:
```
AttributeError: module 'ppb_vector.vector2' has no attribute 'VectorLike'
```
Looks like we need to expose `VectorLike` in `ppb-vector` explicitely?

Reply:
That's not how Python namespaces work.

```
pip list
Package        Version
-------------- -------
atomicwrites   1.3.0  
attrs          19.1.0 
cffi           1.12.2 
dataclasses    0.6    
greenlet       0.4.13 
more-itertools 7.0.0  
pip            19.0.3 
pluggy         0.9.0  
ppb-vector     0.3    
py             1.8.0  
pygame         1.9.5  
pytest         4.4.0  
readline       6.2.4.1
setuptools     40.8.0 
six            1.12.0 
wheel          0.33.1 
```

Reply:
Oh, right, that's the bug with Cirrus being somehow worse at caching than Docker itself.
*needs to report the bug, still.*